bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/TTSService.java

```diff
@@ -72,4 +75,34 @@ public interface TTSService {
      *             {@link AudioStream}
      */
     public AudioStream synthesize(String text, Voice voice, AudioFormat requestedFormat) throws TTSException;
+
+    /**
+     * Check if this TTS service accepts the system-wide TTS cache service.
+     *
+     * @return
+     */
+    public default boolean isCacheEnabled() {
```

Comment:
One example : the TTS service already uses its own cache, better suited, and doesn't want the global one ? Or there is a compatibility issue with it ? In this case, if the cache is defaulted to true, a user installing the TTS service and not aware of this subtlety will complain that it is not working. 

But, let's be clear : I added this parameter because I wanted to let as many options as possible for other (dev and user), and I didn't want to the cache system to force other TTS. But in fact I'm totally in favor of simplifying the code here and letting only one global parameter, if it is deemed solid enough.

Reply:
It can default to `false`, that's not my point. I would prefer a user-configurable option for that, not something in the interface.

Reply:
Done, with a twist I propose : the TTSService is now responsible for using the cache or not.
(see below)